R/defs.bzl

```diff
@@ -1,4 +1,4 @@
-# Copyright 2017 GRAIL, Inc.
+# Copyright 2018 The Bazel R Rules Authors.
```

Comment:
I talked to my team and we have decided to attribute the copyright to Bazel authors. So `Copyright 2018 The Bazel Authors.` will do. Please change everywhere. As for CLA, if you have signed the Google CLA, then we are fine.

Reply:
Done.  I did sign the Google CLA.